apps/site/components/Downloads/DownloadReleasesTable/DetailsButton.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import { useTranslations } from 'next-intl';
+import { use, type FC } from 'react';
+
+import LinkWithArrow from '@/components/LinkWithArrow';
+import { ReleaseModalContext } from '@/providers/releaseModalProvider';
+import type { NodeRelease } from '@/types';
+
+type DetailsButtonProps = {
+  versionData: NodeRelease;
+};
+
+const DetailsButton: FC<DetailsButtonProps> = ({ versionData }) => {
+  const t = useTranslations('components.downloadReleasesTable');
+
+  const { openModal } = use(ReleaseModalContext);
+
+  return (
+    <LinkWithArrow onClick={() => openModal(versionData)}>
```

Comment:
-1 to use icon here because it's always wrap it on Mac/Iphone (device that I had use to test).

<img width="668" src="https://github.com/user-attachments/assets/92d96620-be60-4dd4-bbd7-06a3bcdd1cd4" />